WriteHDLUtils.py

```diff
@@ -778,14 +778,43 @@ def matchArgPortNames_PD(argname, portname, memoryname):
     raise ValueError("DuplicateRemoval is not implemented yet!")
     return False
 
+################################
+# Parse HLS code
+################################
+
+def splitByComma(aString):
+    """
+    # Splits a string by comma delimiter into a list,
+    # but ignores commas that are between balanced angular brackets "<...,...>".
+    """
+    ignore = 0
+    word = ''
```

Comment:
`word` is never used, should it be removed?

Reply:
agreed.